client/src/components/participant-form/Fields.js

```diff
@@ -25,18 +42,13 @@ export const Fields = ({ isDisabled }) => {
 
   return (
     <>
-      <Box mb={2}>
-        <Alert severity="info">
-          <Typography variant="body2" gutterBottom>
-            <b>Need Help?</b><br />
-            Contact a Health Career Access Program agent.
-            <br />
-            <Link href="tel:+18773740463">
-              Dial 1-877-374-0463
-            </Link>
-            <b> Service is available from 8:00 am - 4:30 pm Pacific Time</b>
-          </Typography>
-        </Alert>
+      <Box mb={2} py={1} px={2} display="flex" alignItems="center" className={classes.info}>
```

Comment:
I'm using MUI props here in addition to a class for some other styles. Do we prefer to use all props or all  classes?